explorer/demo/go.mod

```diff
@@ -1,22 +1,18 @@
-module demo
+module demoe_explorer
 
-go 1.19
-
-replace github.com/gioui-plugins/gio-plugins => ../../
```

Comment:
Replace is required, otherwise the `demo` will be based on the github version, instead of the current "under-working" package. Or need to change to [workspaces](https://go.dev/doc/tutorial/workspaces).

Reply:
The first time I used this library, I encountered a problem, the interface of all demos is the same, and the solution for a long time is that the module has the same name